lms/djangoapps/course_home_api/outline/views.py

```diff
@@ -462,11 +462,9 @@ def get(self, request, *args, **kwargs):
             is_masquerading=user_is_masquerading,
         )
         if navigation_sidebar_caching_is_disabled := courseware_disable_navigation_sidebar_blocks_caching():
-            cached = False
-            course_blocks = None
-        else:
             course_blocks = cache.get(cache_key)
-            cached = course_blocks is not None
+        else:
+            course_blocks = None
```

Comment:
Is this conditional reversed? It looks like we're doing the `cache.get` if `navigation_sidebar_caching_is_disabled`? It also looks like `navigation_sidebar_caching_is_disabled` is never referenced later?

Reply:
Oh, thanks for noticing, it was most likely due to a cherry-pick from our repository. Fixed it.